tools/docker-network/docker-compose.yml

```diff
@@ -297,11 +297,6 @@ services:
       --inx.address=node-1-validator:9029
       --faucet.bindAddress=inx-faucet:8091
       --faucet.rateLimit.enabled=false
-      --faucet.baseTokenAmount=1000000000
```

Comment:
Can you undo this change please? We need the higher values, not the default ones.